src/main/interface/cli.c

```diff
@@ -384,6 +384,10 @@ static void cliPrintErrorLinef(const char *format, ...)
     cliPrintLinefeed();
 }
 
+static void cliPrintCorruptMessage(int value)
+{
+    cliPrintf("%d ###CORRUPTED CONFIG###", value);
```

Comment:
I like the idea of consistency so we can use a pattern to match and decorate errors on the configurator side as we've been discussing.  But it's difficult to mix that in here in a way that also conveys the "corruption" aspect.  Particularly since this function is called downstream of other output preceding it on the same line.

Thinking about the separate error detection/decorating parsing in the configurator CLI output maybe we just make the decision that the string "###" is the trigger and reserve that for only reporting errors.  That would work with the current `###ERROR###` indicator as well as this `###CORRUPTED CONFIG###` message.

Would need to add some validation to the few text input values (`name`, `manufacturer_id`) but those are already vulnerable to somebody putting in `###ERROR###` as the value so the validation is needed as is.